release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -0,0 +1,896 @@
+[discrete]
+[id="ocp-4-12-bare-metal-hardware-bug-fixes"]
+==== Bare Metal Hardware Provisioning
+
+[discrete]
+[id="ocp-4-12-builds-bug-fixes"]
+==== Builds
+
+[discrete]
+[id="ocp-4-12-cloud-compute-bug-fixes"]
+==== Cloud Compute
+
+[discrete]
+[id="ocp-4-12-cluster-version-operator-bug-fixes"]
+==== Cluster Version Operator
+
+[discrete]
+[id="ocp-4-12-console-Metal3-bug-fixes"]
+==== Console Metal3 Plug-in
+
+[discrete]
+[id="ocp-4-12-dns-bug-fixes"]
+==== Domain Name System (DNS)
+
+[discrete]
+[id="ocp-4-12-image-registry-bug-fixes"]
+==== Image Registry
+
+[discrete]
+[id="ocp-4-12-installer-bug-fixes"]
+==== Installer
+
+[discrete]
+[id="ocp-4-12-kube-api-server-bug-fixes"]
+==== Kubernetes API server
+
+[discrete]
+[id="ocp-4-12-kube-scheduler-bug-fixes"]
+==== Kubernetes Scheduler
+
+[discrete]
+[id="ocp-4-12-machine-config-operator-bug-fixes"]
+==== Machine Config Operator
+
+[discrete]
+[id="ocp-4-12-compliance-operator-bug-fixes"]
+==== Compliance Operator
+
+[discrete]
+[id="ocp-4-12-management-console-bug-fixes"]
+==== Management Console
+
+[discrete]
+[id="ocp-4-12-monitoring-bug-fixes"]
+==== Monitoring
+
+[discrete]
+[id="ocp-4-12-networking-bug-fixes"]
+==== Networking
+
+[discrete]
+[id="ocp-4-12-ne-perf-improvements"]
+==== Networking performance improvements
+
+[discrete]
+[id="ocp-4-12-node-bug-fixes"]
+==== Node
+
+[discrete]
+[id="ocp-4-12-openshift-cli-bug-fixes"]
+==== OpenShift CLI (oc)
+
+[discrete]
+[id="ocp-4-12-Kubernetes-controller-manage-bug-fixes"]
+==== Kubernetes Controller Manager
+
+[discrete]
+[id="ocp-4-12-olm-bug-fixes"]
+==== Operator Lifecycle Manager (OLM)
+
+[discrete]
+[id="ocp-4-12-openshift-operator-sdk-bug-fixes"]
+==== Operator SDK
+
+[discrete]
+[id="ocp-4-12-openshift-api-server-bug-fixes"]
+==== OpenShift API server
+
+[discrete]
+[id="ocp-4-12-rhcos-bug-fixes"]
+==== {op-system-first}
+
+[discrete]
+[id="ocp-4-12-performance-addon-operator-bug-fixes"]
+==== Performance Addon Operator
+
+[discrete]
+[id="ocp-4-12-routing-bug-fixes"]
+==== Routing
+
+[discrete]
+[id="ocp-4-12-scalability-and-performance-bug-fixes"]
+==== Scalability and performance
+
+[discrete]
+[id="ocp-4-12-storage-bug-fixes"]
+==== Storage
+
+[discrete]
+[id="ocp-4-12-web-console-developer-perspective-bug-fixes"]
+==== Web console (Developer perspective)
+
+[id="ocp-4-12-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+* *TP*: _Technology Preview_
+* *GA*: _General Availability_
+* *-*: _Not Available_
+* *DEP*: _Deprecated_
+
+//TODO: remove anything that has been GA since 4.7?
+
+.Technology Preview tracker
```

Comment:
Were you going to update the framework of this table to support 4.12?

Reply:
I have updated this as a draft for 4.12. The table now supports 4.10, 4.11, and 4.12.